README.md

```diff
@@ -107,6 +106,24 @@ steps:
   run: ls -R
 ```
 
+# Download path output
+
+Not sure where your artifact was downloaded to? The `download-path` step output can help you out. Just make sure to be aware of the extra directory that is created if downloading all artifacts (no name specified).
+
+```yaml
+steps:
+- uses: actions/checkout@v2
+
+- uses: actions/download-artifact@v2
+  id: download
+  with:
+    name: 'my-artifact'
+    path: path/to/artifacts
+
+- name: 'Output download path'
+  run: echo ${{steps.download.outputs.download-path}}
```

Comment:
In examples that use step outputs, I like to make it clear that `download` here matches the `id` set in the download-artifact step, rather than being some fixed constant.

For example: https://github.com/actions/cache#skipping-steps-based-on-cache-hit
>Note: The id defined in actions/cache must match the id in the if statement (i.e. steps.[ID].outputs.cache-hit)